packages/apptive_grid_form/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 0.10.0-alpha.4
 * Update Core Package to 0.10.0-alpha.4 to include breaking changes to GridField
+* Add `description` to forms
```

Comment:
Same here. This version is already released so please bump the version to -alpha.5

Reply:
done